sources/packages/backend/apps/db-migrations/src/migrations/1706733349802-RemoveConstraintStudentAssessmentsOnlyOneAssessmentDataNullIndex.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class RemoveConstraintStudentAssessmentsOnlyOneAssessmentDataNullIndex1706733349802
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Drop-index-only-one-assessment-data-null.sql",
```

Comment:
DEVS, did we decide it's ok to re-use existing migration sql files?

Reply:
New files are created in my latest commit. Please have a look.